hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,20 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
```

Comment:
Please make these relative links, rather than absolute URLs.

Reply:
Not needed - the way this page is found (excluding external links) is via the boot modes page.

If an external link is used, the page includes "breadcrumbs" which the user can use to go up the tree to the boot modes page.

Reply:
The reason I prefer relative links, is that it makes the documentation pages equally readable on https://github.com/raspberrypi/documentation/ and https://www.raspberrypi.org/documentation/